library/network/src/modules/SuSEFirewallServices.rb

```diff
@@ -752,6 +752,24 @@ def GetPossiblyConflictServices
   end
 
   class SuSEFirewalldServicesClass < SuSEFirewallServicesClass
+    SERVICES_DIR = ["/etc/firewalld/services", "/usr/lib/firewalld/services"]
```

Comment:
- this should be named `SERVICE_DIRS`
- it is only used in one place, in a test. that is suspicious.